TransferMethod/Sources/AddTransferMethodController.swift

```diff
@@ -96,6 +95,11 @@ final class AddTransferMethodController: UITableViewController {
         titleDisplayMode(.always, for: presenter.transferMethodTypeCode.lowercased().localized())
     }
 
+    // MARK: - Setup Layout -
+    private func setupLayout() {
+        setupTableView()
+    }
+
```

Comment:
What is this function?

Reply:
This is weird,  I didn't add this method. Its already there in the code (Authors view shows Debojit added it last year)
Not sure why GitHub is showing as new code

Reply:
just remove it, if not being used as setupTableView() is already being called from viewDidLoad